<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TaskTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>17</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>待做事项</string>
    </property>
   </widget>
   <widget class="QLabel" name="Article">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>257</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>待读论文</string>
    </property>
   </widget>
   <widget class="QToolButton" name="AddArticle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Record">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>471</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>记笔记</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ArticleContent1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ArticleName1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ArticleName2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ArticleContent2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Time1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Time5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DoneTask1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DoneTask2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DoneTask3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DoneTask4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DoneTask5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DoneTask6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Task5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="OpenArticle1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>跳转</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ReadArticle1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>已读</string>
    </property>
   </widget>
   <widget class="QToolButton" name="OpenArticle2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>跳转</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ReadArticle2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>已读</string>
    </property>
   </widget>
   <widget class="QToolButton" name="AddTask">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddArticle">
   <property name="text">
    <string>AddArticle</string>
   </property>
  </action>
  <action name="actionAddTask">
   <property name="text">
    <string>AddTask</string>
   </property>
  </action>
  <action name="actionDoneTask1">
   <property name="text">
    <string>DoneTask1</string>
   </property>
  </action>
  <action name="actionDoneTask2">
   <property name="text">
    <string>DoneTask2</string>
   </property>
  </action>
  <action name="actionDoneTask3">
   <property name="text">
    <string>DoneTask3</string>
   </property>
  </action>
  <action name="actionDoneTask4">
   <property name="text">
    <string>DoneTask4</string>
   </property>
  </action>
  <action name="actionDoneTask5">
   <property name="text">
    <string>DoneTask5</string>
   </property>
  </action>
  <action name="actionDoneTask6">
   <property name="text">
    <string>DoneTask6</string>
   </property>
  </action>
  <action name="actionOpenArticle1">
   <property name="text">
    <string>OpenArticle1</string>
   </property>
  </action>
  <action name="actionOpenArticle2">
   <property name="text">
    <string>OpenArticle2</string>
   </property>
  </action>
  <action name="actionReadArticle1">
   <property name="text">
    <string>ReadArticle1</string>
   </property>
  </action>
  <action name="actionReadArticle2">
   <property name="text">
    <string>ReadArticle2</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
